dina-test-support/src/test/resources/fieldThatShouldNotExist.json

```diff
@@ -18,6 +18,12 @@
       "description":{
         "en":"obj",
         "fr":"obj-fr"
+      }, 
+      "customObject": {
+        "name": {
+          "firstName": "firstName"
+        },
+        "id": 0
```

Comment:
This was added since this test is designed for this part:

```json
"fieldShouldNotExist": "field",
```

Not the required fields.